### DISCUSSION TITLE
Climate Configuration

### DISCUSSION THREAD

**Discussion by SebastianDietl** [Blackymas/NSPanel_HA_Blueprint#1943] [General]
Hi, 
how do i get this view? When i configure climate i get a horizontal bar
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/50870031/c602b50c-2615-4663-a71e-8de00a5f679d)

**Comment by edwardtfn** [comment]
This page was replaced by a new one with a slider. This was a while ago.
That old format was quite buggy with the limitations imposed by the display model used by Sonoff on these panels and was also blocking the dual target climate.

**Reply by SebastianDietl** [reply]
ok thank you - so the slider is the new one? for me the circle looks better... sad that it did not work

**Reply by edwardtfn** [reply]
Indeed. The circle was looking much nicer. 
Perhaps we can try it again in the near future, 😉

**Comment by DerBo19** [comment]
Dear @edwardtfn,

you've helped me so many times getting my panels up and running in our new house. Thank you for that. We are now quite satisfied. Due to the many other topics, I'm a bit out of touch with the subject and would actually like to control the underfloor heating with the panels. Unfortunately I can't get the control to be selected as an entity in the blueprint. I've tried it before directly via relays on the panel and it worked. But now i would like to use a sonoff 4CHPROR3 as relays.
I would like to use the sensor of the panel in the room as the sensor (e.g. sensor.nspanelbaddg_temperature). The valves are controlled via the Sonoff 4CHPROR3 flashed with tasmota. This is already integrated into HA. Unfortunately, when I click on the temperature on the panel, the corresponding page remains blank and I only see the slider. What am I doing wrong? Can you help me again? Thank you very much, as always!

**Reply by DerBo19** [reply]
OK guys, got it! Like a bloody beginner i only restartet the yaml after configuring everything instead of restart the complete HA. After that, the climate-entity appeared. Think i got it now. Anyway thanks again for all the support. BTW: i also preffered the circle ;)
